seq2science/rules/gene_counts.smk

```diff
@@ -67,6 +67,8 @@ if config["quantifier"] == "salmon" and config["tpm2counts"] == "tximeta":
             cts=get_counts,
         output:
             counts=expand("{counts_dir}/{{assembly}}-counts.tsv", **config),
+            tpms=expand("{counts_dir}/{{assembly}}-TPM.tsv",**config),
+            lengths=expand("{counts_dir}/{{assembly}}-gene_lengths.tsv",**config),
```

Comment:
unrelated to the PR, but maybe there should be an input requirement for that the assembly is from ensembl? An input function should be able to do this right?

Reply:
Maybe I should make an issue about it though, instead of puitting it here